src/injectify.js

```diff
@@ -12,7 +12,12 @@ function processRequireCall(path) {
 }
 
 export default function injectify(context, source, inputSourceMap) {
-  const { ast } = transform(source);
+  const { ast } = transform(source, {
+    babelrc: false,
+    code: false,
```

Comment:
while digging through the Babel source code, I came across [this check](https://github.com/babel/babel/blob/v6.24.1/packages/babel-core/src/transformation/file/index.js#L602). Since we only care about `ast` in this step, we can prevent Babel from doing further work by passing `code: false` here.